### DISCUSSION TITLE
Run on Linux/Debian 12?

### DISCUSSION THREAD

**Discussion by Stan-Green** [ptrumpis/snap-camera-server#173] [Q&A] [answered]
From looking at the source code, it looks like this server was written on Linux. The fact that it runs on a Mac also points to a Linux platform. So, has anyone run this on Linux/Debian 12? What about a Raspberry Pi? I want to run it on a server on my network to use the client on multiple computers without installing it on each. (I have many Linux servers available.)

**Comment by ptrumpis** [comment, marked as answer]
This user guide might help:
[Setup instructions for Linux users #105](https://github.com/ptrumpis/snap-camera-server/discussions/105)
